clients/p4rt-ctl/p4rt-ctl.in

```diff
@@ -44,6 +44,8 @@ from functools import wraps
 import google.protobuf.text_format
 from google.rpc import status_pb2, code_pb2
 
+from idpf import p4info_pb2 as idpf_p4info
```

Comment:
1. I would strongly recommend holding off on changing p4rt-ctl until there's an actual need for the idpf modules. If p4rt-ctl lacks the logic to use the new fields, we have _no business_ penalizing the user. That's just rude.
2. When we do introduce them, the imports should be done conditionally (iirc, you enclose them in a try/catch block).
   - If the imports succeed, set a flag indicating that we have them.
   - If the imports fail, do nothing. Or perhaps issue a simple informational message that "the idpf extensions are unavailable". 
3. If the user issues a request that requires the idpf extensions, we check the flag. If it's false, we issue an error message.

We have every reason to expect that users won't have installed the updated p4runtime wheel. They should continue to be able to use `p4rt-ctl`, without being disadvantaged, until they have an actual need for the new/changed functionality.

At that time, we issue a polite error message explaining that the extensions are not available, and advise the user that they need to update the python p4runtime module in order to perform the operation.

Reply:
have put the new imports in try/catch block.